modules_core/com.etendorx.das/src/main/java/com/etendorx/das/utils/DefaultFilters.java

```diff
@@ -21,42 +21,49 @@ public class DefaultFilters {
 
 
     public static String addFilters(String sql, String userId, String clientId, String orgId, String isActive) {
-            //SUPERUSER BY PASS FILTERS
-            if (isSuperUser(userId, clientId, orgId)) {
-                return sql;
-            }
+        //AUTH SERVICE BY PASS FILTERS
+        if (isAuthService(userId, clientId, orgId)) {
+            return sql;
+        }
+        //SUPERUSER BY PASS FILTERS
+        if (isSuperUser(userId, clientId, orgId)) {
+            return sql;
+        }
 
-            boolean containsWhere = sql.contains(WHERE);
-            //GET THE TABLE NAME USING REGULAR EXPRESSION
-            String tableAlias = getTableAlias(sql);
+        boolean containsWhere = sql.contains(WHERE);
+        //GET THE TABLE NAME USING REGULAR EXPRESSION
+        String tableAlias = getTableAlias(sql);
 
-            //RETURN DEFAULT FILTERS
-            return replaceInQuery(sql, clientId, orgId, isActive, containsWhere, tableAlias);
+        //RETURN DEFAULT FILTERS
+        return replaceInQuery(sql, clientId, orgId, isActive, containsWhere, tableAlias);
     }
 
     @NotNull
     private static String replaceInQuery(String sql, String clientId, String orgId, String isActive, boolean containsWhere, String tableAlias) {
         String resultSql;
         if (containsWhere) {
             resultSql = sql.replace(WHERE, WHERE + tableAlias +".ad_client_id = '"+ clientId +"' " +
-                    "and "+ tableAlias +".ad_org_id = ANY (etrx_org_tree('"+ orgId +"')) " +
-                    "and "+ tableAlias +".isactive = '"+ isActive +"' and ");
+                    "AND (" + tableAlias + ".ad_org_id =  '"+ orgId +"' OR ((etrx_is_org_in_org_tree(" + tableAlias + ".ad_org_id, '"+ orgId +"', '1')) = 1))" +
```

Comment:
Using sugar syntax this can do it in a single line as:

```
private static String replaceInQuery(String sql, String clientId, String orgId, String isActive, boolean containsWhere, String tableAlias) {
  return sql.replace((containsWhere? WHERE:LIMIT), WHERE + tableAlias +".ad_client_id = '"+ clientId +"' " +
"AND (" + tableAlias + ".ad_org_id =  '"+ orgId +"' OR ((etrx_is_org_in_org_tree(" + tableAlias + ".ad_org_id, '"+ orgId +"', '1')) = 1))" +
"AND "+ tableAlias +".isactive = '"+ isActive + (containsWhere? "' AND ":"' limit ") );
}
```